src/drive/components/FolderView.jsx

```diff
@@ -113,6 +119,15 @@ class FolderView extends Component {
       </Main>
     )
   }
+
+  renderViewer(children) {
```

Comment:
Sort of secondary, but why is this called `renderViewer `? Is there a viewer involved?

Reply:
As the viewer is instantiated by a child route, it is passed to this component as a child and the `renderViewer` method passes the `files` prop required by the viewer. I'm not sure I'm clear here...

Reply:
Yeah ok I understand now, but the code doesn't make it obvious. Since technically the `children` could be anything, I think we shouldn't refer to a viewer here. Or maybe adding a comment about what's going on.

Reply:
I'll add a comment to make it clearer, but tbh, I'm not fan of this trick (that we used in Photos before) and I'd like a better solution, but it'd require a lot of work